src/org/mozilla/javascript/NativeJavaList.java

```diff
@@ -9,9 +9,17 @@
 import java.util.List;
 
 public class NativeJavaList extends NativeJavaObject {
+    private static final long serialVersionUID = 1L;
 
     private List<Object> list;
 
+    static void init(ScriptableObject scope, boolean sealed) {
+        NativeJavaList obj = new NativeJavaList();
+        obj.exportAsJSClass(scope, "JavaObject", sealed);
```

Comment:
Why these three? There are many interfaces and classes in Java. For example, if you want to extend a `Set` or `Stream` with a prototype, do you have to add more JavaXXX each time? Why `List` instead of `Collection`?

If you have a problem you want to solve, it is better to fully discuss what specifications you want to use before implementing them.